web/interfaces.js

```diff
@@ -149,7 +155,10 @@ class IRenderableView {
    */
   draw() {}
 
-  resume() {}
+  /**
+   * @type {?(() => void)}
+   */
+  get resume() {}
```

Comment:
This also looks somewhat weird, since when `resume` is defined it's a regular function and *not* a getter.

Reply:
Agree, but in the implementing classes (e.g. `PDFPageView`) , it's actually initialized to be `null`

https://github.com/mozilla/pdf.js/blob/bf5a45ce6d0e9d070204859a39844bcdaf69e9aa/web/pdf_page_view.js#L116

and later set to a function

https://github.com/mozilla/pdf.js/blob/bf5a45ce6d0e9d070204859a39844bcdaf69e9aa/web/pdf_page_view.js#L571

which causes the generated types for `PDFPageView` and this interface to disagree unless I type `resume` like this. I felt like it'd be safer for this PR to have the types reflect the current implementation rather than change the implementation to match the desired types.

Reply:
> which causes the generated types for `PDFPageView` and this interface to disagree unless I type `resume` like this.

But this actually means the interface-definition is now by definition *incorrect*, which really doesn't help someone trying to see how it should be correctly implemented. Can't we just tag the method as follows instead?
```javascript
/**
 * @ignore
 */
resume() {}
```

Reply:
`tsc` seems to be ignoring the `@ignore` annotation and generates types anyway, so I either have to set the type to describe the current situation or fix the underlying implementations to match the desired type of `() => void`. I'll make an attempt at a fix since it looks relatively simple